theme/size.go

```diff
@@ -134,6 +139,13 @@ func SelectionRadiusSize() float32 {
 	return Current().Size(SizeNameSelectionRadius)
 }
 
+// ScrollBarRadiusSize returns the scroll bar corner radius size.
+//
+// Since: 2.5
+func ScrollBarRadiusSize() float32 {
```

Comment:
actually we should probably not add this since we're deprecating top-level theme accessors?

Reply:
Good point

I felt bad about all the theme merge conflicts so I was just tidying this up so it could merge. I will delete that too.